cmd/podman/play_kube.go

```diff
@@ -299,6 +304,9 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 		if err := shared.ValidateVolumeCtrDir(volume.MountPath); err != nil {
 			return nil, errors.Wrapf(err, "error in parsing MountPath")
 		}
+		// At this point, we know the name of tho host and the container path, but we don't know the SElinux MountLabel of the container.
+		// Therefore, we can't set the permissions of the mount here. Instead, mark the mount as shared (sharedMountPermission),
+		// and allow libpod to correctly set the permissions once the container is created
 		containerConfig.Volumes = append(containerConfig.Volumes, fmt.Sprintf("%s:%s", host_path, volume.MountPath))
```

Comment:
I'm not seeing sharedMountPermission anywhere here, though?

Reply:
oof out of date comment